packages/css-framework/src/_config.scss

```diff
@@ -1,17 +1,25 @@
-$navy--font-size: 16px !default;
-$navy--font-family: "lato" !default;
-$navy--context: "light" !default;
+/*
+
+  These config variables can be overridden by the 
+  consumer when they import the CSS Framework.
+
+*/
+
+$font-size:   16px !default;
+$font-family: "lato" !default;
+$context:     "light" !default;
 
 // Contexts
-$navy--breakpoints: () !default;
-$navy--colors: () !default;
-$navy--spacing: () !default;
-$navy--shadows: () !default;
-$navy--typography: () !default;
-$navy--zIndex: () !default;
+$breakpoints: () !default;
+$colors:      () !default;
+$spacing:     () !default;
+$shadows:     () !default;
+$typography:  () !default;
+$zindex:      () !default;
 
 
 // Utility Namespace
 $utility-ns: "rn_";
 
-$navy-content-width: 1280px;
+$content-width: 1280px;
+$animation-timing: 0.3s;
```

Comment:
Regarding missing newlines at end of files - depending on what is parsing / processing these files it can cause problems. My editor adds them on save automatically just to be safe.

https://unix.stackexchange.com/a/18789

>A text file, under unix, consists of a series of lines, each of which ends with a newline character (\n). A file that is not empty and does not end with a newline is therefore not a text file.
>
>Utilities that are supposed to operate on text files may not cope well with files that don't end with a newline; historical Unix utilities might ignore the text after the last newline, for example. GNU utilities have a policy of behaving decently with non-text files, and so do most other modern utilities, but you may still encounter odd behavior with files that are missing a final newline¹.